tests/async_client_test.py

```diff
@@ -749,6 +749,8 @@ async def test_sync(self, async_client, aioresponse):
         # Test with filter dict
 
         aioresponse.get(
+            # TODO: what the hell is the difference here?
+            #f'https://example.org/_matrix/client/r0/sync?access_token=abc123&since=s526_47314_0_7_1_1_1_11444_1&filter={quote("{}")}',
             re.compile(url + r"&filter=" + quote("{}") + "&since=.*"),
```

Comment:
This is the test that isn't passing; the string above will work just fine, however.